1 Niagara 4 Overview Exercise - BFile/myFile-rt/module.palette

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <bajaObjectGraph version="4.0" reversibleEncodingKeySource="none" FIPSEnabled="false" reversibleEncodingValidator="[null.1]=" >
 <p m="b=baja" t="b:Folder">
+<p m="dtl=myFile" t="dtl:File" n="File" />
```

Comment:
There's nothing wrong with using `dtl` as the symbol for `myFile`, but it's not very descriptive. It might make more sense to use something like `mf`:

```xml
<p m="mf=myFile" t="mf:File" n="File" />
```